feature-libs/pickup-in-store/components/container/pickup-option-dialog/pickup-option-dialog.component.ts

```diff
@@ -47,19 +49,28 @@ export class PickupOptionDialogComponent implements OnInit, OnDestroy {
   cartId: string;
   userId: string;
 
-  readonly focusConfig: FocusConfig = {
-    trap: true,
-    block: true,
-    autofocus: 'input',
-    focusOnEscape: true,
-  };
-
   readonly ICON_TYPE = ICON_TYPE;
   /** The reason given closing the dialog window without selecting a location */
   readonly CLOSE_WITHOUT_SELECTION = 'CLOSE_WITHOUT_SELECTION';
   /** The reason given closing the dialog window after selecting a location */
   readonly LOCATION_SELECTED = 'LOCATION_SELECTED';
 
+  protected featureConfigService = inject(FeatureConfigService);
```

Comment:
We can make the featureConfigService `private` to make it easier to clean up. This is our "use of private" exception ;)

Reply:
Sure!